CONTRIBUTING.md

```diff
@@ -690,6 +690,45 @@ query localConnections {
 curl -s -H "Content-Type:application/json" -H "Authorization: Bearer ${DTX_ACCESS_TOKEN}" http://localhost:26636/gateway/v1/graphql/schema.graphql
 ```
 
+### Invoking the Control Plane Proxy API
+
+The Control Plane Proxy API allows you to make calls to the Confluent Cloud control plane, specifically for artifact and compute pool management endpoints defined by the configuration:
+
+```yaml
+ccloud-api-control-plane-regex: "(/artifact.*)|(/fcpm/v2/compute-pools.*)"
+```
+
+Control Plane endpoints:
+  http://localhost:26636/artifact/* (For artifact management)
+  http://localhost:26636/fcpm/v2/compute-pools/* (For Flink compute pools management)
+
+Required headers:
+
+- Authorization: Bearer ${DTX_ACCESS_TOKEN}
+- x-connection-id: <connection-id> (Must reference a valid Confluent Cloud connection)
```

Comment:
Should we say "authenticated" Confluent Cloud connection? Otherwise, the connection won't hold a (valid) control plane token.

```suggestion
- x-connection-id: <connection-id> (Must reference an authenticated Confluent Cloud connection)
```